.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: "Build"
+on:
+  pull_request:
+  push:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - run: |
+        sudo apt-get update
+        sudo apt-get install --yes curl libsndfile1-dev portaudio19-dev
+    - uses: actions/cache@v1
+      with:
+        path: ~/.stack
+        key: ${{ runner.os }}-stack
+    - uses: actions/checkout@v2
+      with:
+        repository: 'soenkehahn/looper'
+        path: 'looper'
```

Comment:
Can we pin the checkout of looper to the current commit?

Reply:
Done.